### ISSUE TITLE
MVN build fails in AzDO pipeline.

### ISSUE THREAD

**Issue by stevegroom** [cloudfoundry-incubator/multiapps-controller#534]
### Description
MVN build fails.

I cloned the repo to azure and created a pipeline to build the application. failed:

```bash
[command]/usr/share/apache-maven-3.6.1/bin/mvn -f /home/vsts/work/1/s/pom.xml -e clean install

[ERROR] Failed to execute goal on project com.sap.cloud.lm.sl.cf.parent: 
Could not resolve dependencies for project com.sap.cloud.lm.sl.cf:com.sap.cloud.lm.sl.cf.parent:pom:1.78.0-SNAPSHOT: 
Could not find artifact com.sap.cloud.lm.sl:com.sap.cloud.lm.sl.common:jar:tests:1.71.0-SNAPSHOT -> [Help 1]

```
### Your environment
*	MultiApps Controller version - --latest--
*	which CF vendor is used - --SAP-- 
*	which backing services are used - n/a

### Steps to reproduce
Clone this repo and add an azure-pipelines.yml file.

```yaml
# Maven
# Build your Java project and run tests with Apache Maven.
# Add steps that analyze code, save build artifacts, deploy, and more:
# https://docs.microsoft.com/azure/devops/pipelines/languages/java

trigger:
- master

pool:
  vmImage: 'ubuntu-latest'

steps:
- task: Maven@3
  inputs:
    mavenPomFile: 'pom.xml'
    mavenOptions: '-Xmx3072m'
    javaHomeOption: 'JDKVersion'
    jdkVersionOption: '1.8'
    jdkArchitectureOption: 'x64'
    publishJUnitResults: false
    testResultsFiles: '**/surefire-reports/TEST-*.xml'
    goals: '-e clean install'

- task: CloudFoundryCLIInstall@0
  displayName: Install Cloud Foundry cli
  inputs:
    cfVersion: '6.43.0'

- task: CloudFoundry@1
  displayName: Push the app to SCP CF LAB_EU10:Basis
  inputs:
    cfEndpoint: '8c50800a-9d51-4be7-9d7b-2162ce9d98dc'
    org: 'LAB_EU10_Shared'
    space: 'basis'
    cfManifest: 'com.sap.cloud.lm.sl.cf.web/target/manifests/manifest.yml'
```

maven 3

### Additional information

The maven environment.

```bash
command]/usr/share/apache-maven-3.6.1/bin/mvn -version
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T19:00:29Z)
Maven home: /usr/share/apache-maven-3.6.1
Java version: 1.8.0_212, vendor: Azul Systems, Inc., runtime: /usr/lib/jvm/zulu-8-azure-amd64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-1045-azure", arch: "amd64", family: "unix"
SYSTEMVSSCONNECTION exists true
[command]/usr/share/apache-maven-3.6.1/bin/mvn -f /home/vsts/work/1/s/pom.xml -e clean install
```

**Comment by ddonchev** [comment]
Hi Steve,
Thanks for reporting this! Looking at the oss travis build, this artifact is downloaded as: 
> [INFO] Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/com/sap/cloud/lm/sl/com.sap.cloud.lm.sl.common/1.72.0-SNAPSHOT/maven-metadata.xml
For reference: https://travis-ci.org/cloudfoundry-incubator/multiapps-controller
I can also reproduce the issue with a just installed maven with no additional configurations on an ubuntu 18.04 . Probably some additional configurations to the pom would need to be added - some which travis already has.
Until those are defined and in case this helps, you can find the "released" versions of the multiapps-project at : https://repo1.maven.org/maven2/com/sap/cloud/lm/sl/ * . The war you'd need to run the backend component is at: https://repo1.maven.org/maven2/com/sap/cloud/lm/sl/cf/com.sap.cloud.lm.sl.cf.web/
and the client cf cli plugin can be installed as `cf install-plugin multiapps`

**Comment by ddonchev** [comment]
Adding the following repository reference fixed the build for the ubuntu machine.
It can be included in the ~/.m2/settings.xml and in the project pom.xml
>    <repositories>
>         <repository>
>             <id>central-sonatype</id>
>             <url>https://oss.sonatype.org/content/groups/public/</url>
>         </repository>
>     </repositories>
By the way, the team does not follow a strict policy on referencing released or snapshot versions of the multiapps project for the master(development) branch. 
If you'd like to use a stable version of the tool, a good idea is to directly use the ones released on nexus or to build from the latest rel branch.
If on the other hand you'd like to have CI running and validating the changes in master, than it would make sense to have a build of the multiapps project too - to have a fresh snapshot version of it's master used for the currently built one.

**Comment by ddonchev** [comment]
fixed with PR #558
